data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/ImportCertificateRpcRequest.avsc

```diff
@@ -11,7 +11,7 @@
     },
     {
       "name": "certificate",
```

Comment:
What is the intention here? A string of concatenated certs? Or should it be an array of Strings? If the comment is changing the plural maybe the name should also? i.e. `certificate` -> `certificates`

Reply:
It should be a string of concatenated certificates.